frontend/src/components/TextDifferenceChecker.jsx

```
@@ -19,74 +21,96 @@ const TextDifferenceChecker = () => {
   };
 
   return (
     <div className="container mx-auto p-4">
       <h2 className="text-2xl font-bold mb-4">Text Difference Checker</h2>
       <div className="grid grid-cols-1 md:grid-cols-2 gap-4 mb-4">
         <div>
-          <label className="block mb-2 text-sm font-medium text-black">Text 1</label>
+          <label className="block mb-2 text-sm font-medium text-foreground">
+            Text 1
+          </label>
           <textarea
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)